packages/blade/src/components/Accordion/AccordionButton.web.tsx

```diff
@@ -36,6 +36,8 @@ const AccordionButton = ({ index, icon: Icon, children }: AccordionButtonProps):
     console.warn(`[Blade: Accordion]: showNumberPrefix and icon shouldn't be used together`);
   }
 
+  const isButtonExpanded = expandedIndex === index;
```

Comment:
Refactored in #1296 to `isItemExpanded`. Had originally used this based on `aria-expanded` which is passed to the button because it's used as a control here 😅